### PR TITLE
[DOCFIX] Add Chinese Catalog.md Doc

### PR DESCRIPTION
**Pull request by thanosaw**

### CODE REVIEW

File: docs/cn/core-services/Catalog.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+## 概述
+
+Alluxio 2.1.0引入了一项Alluxio目录服务的新Alluxio服务。
+Alluxio目录是用于管理对结构化数据访问的服务，
+所提供服务类似于[Apache Hive Metastore](https://cwiki.apache.org/confluence/display/Hive/Design#Design-Metastore)
+
+SQL Presto，SparkSQL和Hive等引擎利用这些类似于元数据存储的
+服务来确定执行查询时要读取哪些数据以及读取多少数据。
+Catalog中存储有关不同数据库目录，表，存储格式，数据
+位置等及更多信息。
```
Comment (LuQQiu) [inline docs/cn/core-services/Catalog.md:21]: One line

File: docs/cn/core-services/Catalog.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+Alluxio目录服务旨在使为Presto查询引擎提供结构化表元数据检索和服务变得更简单和直接，例如[PrestoSQL](https://prestosql.io/)，
+[PrestoDB](https://prestodb.io)和[Starburst Presto](https://starburstdata.com)。
+
+
```
Comment (LuQQiu) [inline docs/cn/core-services/Catalog.md:25]: extra empty line

File: docs/cn/core-services/Catalog.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+##系统架构
+
+Alluxio目录服务的设计与普通的Alluxio文件系统非常相似。
+服务本身并不负责保有所有数据，而是作为一个源自另一个位置(如MySQL，Hive)的
+元数据缓存服务。
+这些被称为**UDB**(**U**nder**D**ata**B**ase)。
```
Comment (LuQQiu) [inline docs/cn/core-services/Catalog.md:31]: Space between **U**nder and *D**ata**B**ase

File: docs/cn/core-services/Catalog.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+###探索附加的数据库
+
+数据库一旦附加，通过`alluxio table ls`检查已挂载数据库
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table ls
+alluxio_db
+```
+
+通过'alluxio tables ls <db_name>`列出数据库下所有表。
```
Comment (LuQQiu) [inline docs/cn/core-services/Catalog.md:107]: backstick before alluxio

File: docs/cn/core-services/Catalog.md [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+如果相应hive中数据库任何表存在，
```
Comment (LuQQiu) [inline docs/cn/core-services/Catalog.md:108]: 相应hive数据库中存在的所有表都会通过此命令列出

File: docs/cn/core-services/Catalog.md [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+通过执行此命令将会被列出。
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table ls alluxio_db
+test
+```
+
+在上述情况下，alluxio_db有一个表test。要获取数据库中表的更多信息，运行
```
Comment (LuQQiu) [inline docs/cn/core-services/Catalog.md:116]: add backsticks around alluxio_db and test

File: docs/cn/core-services/Catalog.md [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+```
+
+运行上述命令之后再运行`alluxio table ls`将不会再显示已分离的数据库。
+
+###同步数据库
+
+当原始底层数据库和表更新后，用户可以调用sync命令
+来刷新存储在Alluxio目录元数据中的相应信息。
+有关更多详细信息参见[sync命令文档]({{ '/en/operation/User-CLI.html' | relativize_url}}＃sync)
+。
```
Comment (LuQQiu) [inline docs/cn/core-services/Catalog.md:188]: put the stop sign in previous line

File: docs/cn/core-services/Catalog.md [hunk 8]
```diff
@@ -0,0 +1,281 @@
...
+`/etc/catalog/catalog_alluxio.properties`
+```properties
+connector.name=hive-alluxio
+hive.metastore=alluxio
+hive.metastore.alluxio.master.address=HOSTNAME:PORT
+```
+
+创建`catalog_alluxio.properties`文件，意味着一个名为`catalog_alluxio`新的目录
+添加到Presto。
+设置`connector.name = hive-alluxio`会将连接器类型设置为Presto新Alluxio连接器名称，即`hive-alluxio`。
```
Comment (LuQQiu) [inline docs/cn/core-services/Catalog.md:243]: remove extra spaces before and after =
the same for the whole graph

**Comment by apc999** [comment]
@thanosaw can you help address comments?

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**